lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -35,7 +35,6 @@ import { TokenResponse } from "./RequestInfo";
 import { User } from "./User";
```

Comment:
This file is essentially a revert-back, correct?

Reply:
Yes that is correct. UserAgentApp changes are coming in a future PR.